modules/gdscript/doc_classes/GDScriptSyntaxHighlighter.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="GDScriptSyntaxHighlighter" inherits="EditorSyntaxHighlighter" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
+	<brief_description>
+		GDScript syntax highlighter class.
```

Comment:
I think the description can be improved, please see [the docs](https://docs.godotengine.org/en/stable/contributing/documentation/docs_writing_guidelines.html#give-an-overview-of-the-node-in-the-brief-description).